source/ScrollVar.h

```diff
@@ -28,10 +28,13 @@ template <typename T>
 class ScrollVar: public Animate<T>
```

Comment:
Oh no.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/c7e90bf4-1c11-4684-9897-01957cd078a2)

Reply:
It looks like this is due to the EndlessSkyLib being build with -fno-rtti, but the unit tests being build with rtti enabled.